tvb_contrib/tvb/contrib/scripts/utils/data_structures_utils.py

```diff
@@ -288,7 +288,7 @@ def list_of_dicts_to_dict_of_lists(lst):
 
 
 def list_of_dicts_to_dicts_of_ndarrays(lst, shape=None):
```

Comment:
docstrings please, with [doctest](https://docs.python.org/3/library/doctest.html) since that would help a lot to illustrate the intended usage

Reply:
The tvb_contrib scripts module is indeed in need of a lot of things. Currently it is used only by me and TVB-multiscale.
I promise to deal with it at a later stage.